src/stdlib/parse_influxdb.rs

```diff
@@ -0,0 +1,332 @@
+use std::collections::BTreeMap;
+
+use chrono::DateTime;
+use influxdb_line_protocol::{FieldValue, ParsedLine};
+
+use crate::compiler::prelude::*;
+
+fn influxdb_lines_to_metrics(line: ParsedLine) -> Vec<ObjectMap> {
+    let ParsedLine {
+        series,
+        field_set,
+        timestamp,
+    } = line;
+
+    let timestamp = timestamp.map(DateTime::from_timestamp_nanos);
+
+    let tags: Option<ObjectMap> = series.tag_set.as_ref().map(|tags| {
+        tags.iter()
+            .map(|t| (t.0.to_string().into(), t.1.to_string().into()))
+            .collect()
+    });
+
+    field_set
+        .iter()
+        .filter_map(|f| {
+            let mut metric = ObjectMap::new();
+            let measurement = &series.measurement;
+            let field_key = f.0.to_string();
+            let field_value = match f.1 {
+                FieldValue::I64(v) => v as f64,
+                FieldValue::U64(v) => v as f64,
+                FieldValue::F64(v) => v,
+                FieldValue::Boolean(v) => {
+                    if v {
+                        1.0
+                    } else {
+                        0.0
+                    }
+                }
+                // TODO: return none or return Error?
+                FieldValue::String(_) => return None,
```

Comment:
In the Vector implementation, when a field contains an string value, it simply ignores it. I don't know if it's okay to silently ignore or it would be better to return an error. 

In my opinion, I would return an error, but then, the Vector's implementation should be changed also so there is no inconsistency between implementations

Reply:
Hmm, yeah, I also lean towards error since we don't have a metric type in Vector to represent metrics that have a string value. We should update the codec implementation too to be consistent.

Reply:
Opened https://github.com/vectordotdev/vector/issues/21133 to track this in the codec implementation